R/compiler.R

```diff
@@ -15,6 +19,11 @@
 #'
 #' with_build_tools(has_compiler())
 has_compiler <- function(debug = FALSE) {
+  res <- getOption("pkgbuild.has_compiler")
+  if (!is.null(res) && is.logical(res) && length(res) == 1 && (res %in% c(TRUE, FALSE))) {
```

Comment:
Can we error if the option is set to something different than `TRUE` or `FALSE`?